lib/rules/no-unused-refs.js

```diff
@@ -233,6 +233,26 @@ module.exports = {
               refsNode = id.parent
             }
             extractUsedForPattern(refsNode)
+          },
+          CallExpression(exp) {
+            if (
+              exp.callee.name !== 'useTemplateRef' &&
+              exp.callee.name !== 'templateRef'
```

Comment:
I don't think `templateRef` will be provided in Vue 3.5 either. Could you please remove `templateRef` check?

Reply:
Due to the alias support provided by language core for templateRef, I have habitually added it. Now remove it :)